eks-anywhere-nutanix/Addons/Core/Botkube/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
Do we have any change between this botokube deployment and other env deployments of botokube? I am thinking let's just parameterize the channel name, the rest seems to be the same. Channel name can be supplied with the global config-map like `conformance-test-env`.

This can deferred, I am just sharing feedback since it looks like we have duplication.

Reply:
Since we have Channel Name and Cluster Name unique per deployment model which comes from SSM parameter. Its not possible to have single deployment with current framework setup we have. I wil think about this in future iterations of framework upgrade.